src/lib/Microsoft.Fx.Portability/Proxy/ProxyAuthenticationHandler.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Looks like the build is complaining because of the byte-order mark removal. Let me know if you need help resolving that issue. The build wants this as UTF with a BOM.

Reply:
In this part i need help. Maybe i have to do it in a different way...

Reply:
I can help resolve this tomorrow.

Reply:
@brunowarmling, the byte order mark error is not something we can turn off easily. The most straightforward thing would be for you to fix the encoding. Some options for this:
* Open and save in notepad - When you save in notepad, you can specify the encoding in the little box at the bottom. Choose "UTF-8 with BOM".
* Use your editor's functionality for setting the encoding. This depends upon the editor you're using.
  * In VS Code, there is a little option in the status bar to select the encoding before saving.
  * In Visual Studio, there's a drop-down in the save dialog that allows you to save with encoding. When you select this, there is another window that comes up, and you can choose "Unicode (UTF-8 with signature) Codepage 65001".

If you use some other editor, let me know and I'll figure out how to set the encoding. Thanks for helping with this.

Reply:
Actually i coded this at visual studio, but changed the file using the browser here in the github... There's a option to do that.
I copied the code from visual studio and pasted at github editor.

When i get home, i will try to use a git client.

Reply:
Ok, it's done... I used the GitHub Desktop to send the changes.
But, the build is still failing... i can't figure out why.

Reply:
Thanks for your patience here. We're clearly not set up well to give clear feedback on these failures for external PRs. The item failing is another one of the "Style Cop" errors:

> src\lib\Microsoft.Fx.Portability\Proxy\ProxyAuthenticationHandler.cs(48,22): Error SA1513: Closing brace should be followed by blank line

I'm happy to help babysit through any additional errors. Looks like we've got some work to do if we're going to be serious about taking external contributions. Thanks for being the guinea pig here, and sorry for the hassle.